src/libraries/Common/src/Interop/Linux/Interop.Libraries.cs

```diff
@@ -9,7 +9,7 @@ internal static partial class Libraries
         internal const string Liblog = "liblog";
 
         internal const string Odbc32 = "libodbc.so.2";
-        internal const string OpenLdap = "libldap-2.4.so.2";
+        internal const string OpenLdap = "libldap-2.5.so.0";
```

Comment:
Would this also pick up `libldap-2.5.so.0.1.6` for example?

Reply:
It looks like `libldap-2.5.so.0` is not the real library it is linked to `libldap-2.5.so.0.1.6` and uses it.

Reply:
> It looks like libldap-2.5.so.0 is not the real library

It is likely a symbolic link to the specific version. It is a common mechanism on *nix for handling library versions.